packit/upstream.py

```diff
@@ -773,13 +775,17 @@ def koji_build(
         :param nowait: don't wait on build?
         :param koji_target: koji target to pick (see `koji list-targets`)
         :param srpm_path: use selected SRPM for build, not dist-git repo & ref
+        :param stage: should the staging koji instance be used?
         """
         if not koji_target:
             raise PackitException(
                 "koji target needs to be set when building directly from upstream"
             )
         # we can't use fedpkg b/c upstream repo is not dist-git
-        cmd = ["koji", "build"]
+        cmd = ["koji"]
```

Comment:
Would it be difficult to place these commands in Packit config? The default would be `koji build`, and could be changed to `koji -c <path> build`, or anything else, really. This seems to me a more sustainable approach than extending the API arguments each time we want to change how an external tool is triggered (*blink, blink,* `FedPKG` :smiley:).

Reply:
I opened a PR with the proposed changes, #1213 , could you please take a look?